packages/syft/src/syft/custom_worker/runner_k8s.py

```diff
@@ -16,6 +16,8 @@
 from .k8s import get_kr8s_client
 
 JSONPATH_AVAILABLE_REPLICAS = "{.status.availableReplicas}"
+CREATE_POOL_TIMEOUT_SEC = 60
+SCALE_POOL_TIMEOUT_SEC = 60
```

Comment:
Since during Create or scale, the kubelet pulls image from an external registry, 
If people build from our base image , which is around 2 GB, due to slow network connectivity, it might timeout sometimes.
Shall we set a higher value?

Reply:
These are pool create & scale timouts - not build & push timeouts. That value is in parity with the ones we have set in docker.

Regarding the pull, on cloud this should be faster as they'll be using registry in the same resource group/region.

Again, values can be changed as we see how it see how things go when we deploy them.